kafka_consumer/main.go

```diff
@@ -122,6 +125,16 @@ func init() {
 		}
 		kafkaPartitionNum = int32(c)
 	}
+
+	s = upstreamURI.Query().Get("max-message-bytes")
```

Comment:
what difference between max-batch-size and max-message-bytes

Reply:
max-batch-size concerns how many row changes can be packed into a single Kafka message, while max-message-bytes limits how many bytes can be in a single Kafka message.

Reply:
what happened when the MQ is plusar?